pkg/constants/chain.go

```diff
@@ -32,6 +32,7 @@ const (
 	AURORA        Chain = "aurora"
 	MaticSupernet Chain = "matic-supernet"
 	ZKEVM         Chain = "zkevm"
+	MantleTestnet Chain = "mantle-testnet"
```

Comment:
lets follow naming conventions with upper case - ignore supernet.
`MANTLE_TESTNET`

Reply:
change in all places